packages/x-components/src/x-modules/related-prompts/components/related-prompt.vue

```diff
@@ -1,60 +1,45 @@
 <template>
-  <div class="x-related-prompt" data-test="related-prompt">
-    <div class="x-related-prompt__info">
-      <slot name="header" :suggestionText="relatedPrompt.suggestionText">
-        {{ relatedPrompt.suggestionText }}
-      </slot>
-      <slot name="next-queries" :nextQueries="relatedPrompt.nextQueries">
-        <SlidingPanel :resetOnContentChange="false">
-          <div class="x-related-prompt__sliding-panel-content">
-            <button
-              v-for="(nextQuery, index) in relatedPrompt.nextQueries"
-              :key="index"
-              @click="onClick(nextQuery)"
-              :class="[
-                'x-button',
-                { 'x-selected': selectedNextQuery === nextQuery },
-                nextQueryButtonClass
-              ]"
-            >
-              <slot name="next-query" :nextQuery="nextQuery">
-                <span>{{ nextQuery }}</span>
-                <CrossTinyIcon v-if="selectedNextQuery === nextQuery" class="x-icon" />
-                <PlusIcon v-else class="x-icon" />
-              </slot>
-            </button>
-          </div>
-        </SlidingPanel>
-      </slot>
-    </div>
-    <div class="x-related-prompt__query-preview">
-      <slot name="selected-query" :selectedQuery="selectedNextQuery">
-        {{ selectedNextQuery }}
-      </slot>
-    </div>
-  </div>
+  <button
+    @click="toggleSuggestion(index)"
+    class="x-related-prompt__button"
+    :class="[{ 'x-related-prompt-selected__button': isSelected(index) }]"
+  >
+    <slot name="related-prompt-button-info">
+      <div class="x-related-prompt__button-info">
+        <span
+          class="x-typewritter-initial"
+          :class="[{ 'x-typewritter-animation': isVisible }]"
+          :style="{
+            animationDelay: `${index * 0.4 + 0.05}s`,
+            '--suggestion-text-length': relatedPrompt.suggestionText.length
+          }"
+        >
+          {{ relatedPrompt.suggestionText }}
+        </span>
+      </div>
+      <CrossTinyIcon v-if="isSelected(index)" class="x-icon-lg" />
+      <PlusIcon v-else class="x-icon-neutral-80 x-icon-lg" />
```

Comment:
Colors should be customizable for customers

Reply:
yes, it was a mistake, that class should not be there